rpc/doc.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Overview
+// Package rpc provides the underpinnings of requests and responses (server.Request)
+// and dispatching them to the API by defining a request
```

Comment:
```suggestion
// and dispatching them to the API by defining a request.
```

Reply:
I know here you're just using old material I've pointed you to, but wondering if we can make this sentence and the next clearer overall. Who do these requests and responses come from?